src/jvm/test-trace-recorder-integration/org/jetbrains/trace_recorder/integration/TraceDebuggerExamplesTraceRecorderIntegrationTest.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2025 JetBrains s.r.o.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.jetbrains.trace_recorder.integration
+
+import org.junit.Test
+import java.nio.file.Paths
+
+class TraceDebuggerExamplesTraceRecorderIntegrationTest : AbstractTraceRecorderIntegrationTest() {
```

Comment:
Name `TraceDebuggerExamplesTraceRecorderIntegrationTest` looks very confusing.

Reply:
this is a pattern of naming such integration tests -- with the downloaded project name